### PR TITLE
added image collection tiles

### PR DESCRIPTION
**Pull request by Tembac**
Added to the demo the posibility to add a tileset of collection of images.

This is to be used with this pull request:
https://github.com/HaxeFlixel/flixel-addons/pull/224

### CODE REVIEW

File: Editors/TiledEditor/source/TiledLevel.hx [hunk 1]
```diff
@@ -91,21 +96,68 @@ class TiledLevel extends TiledMap
 		}
 	}
 	
-	public function loadObjects(state:PlayState)
+	public function loadObjects()
```
Comment (Gama11) [inline Editors/TiledEditor/source/TiledLevel.hx]: Why remove the `state` parameter here? Seems nicer to pass it than to create a static reference to it that's only needed in a single function.

Comment (Tembac) [inline Editors/TiledEditor/source/TiledLevel.hx]: I'm used to work like that on my games. Is it wrong?

Comment (Gama11) [inline Editors/TiledEditor/source/TiledLevel.hx]: There isn't really a "right" or "wrong"; it just seems unnecessary in this case.

**Comment by Gama11** [comment]
The demo definitely looks a lot nicer now! :+1:

**Comment by Tembac** [comment]
Thanks! I'm glad to contribute in small things!

**Pull request merged**